test/e2e_node/density_test.go

```diff
@@ -533,6 +533,21 @@ func createBatchPodSequential(f *framework.Framework, pods []*v1.Pod) (time.Dura
 	return batchLag, e2eLags
 }
 
+// verifyLatencyWithinThreshold verifies whether 50, 90 and 99th percentiles of a latency metric are
+// within the expected threshold.
+func verifyLatencyWithinThreshold(threshold, actual e2emetrics.LatencyMetric, metricName string) error {
+        if actual.Perc50 > threshold.Perc50 {
+                return fmt.Errorf("too high %v latency 50th percentile: %v", metricName, actual.Perc50)
+        }
+        if actual.Perc90 > threshold.Perc90 {
+                return fmt.Errorf("too high %v latency 90th percentile: %v", metricName, actual.Perc90)
+        }
+        if actual.Perc99 > threshold.Perc99 {
+                return fmt.Errorf("too high %v latency 99th percentile: %v", metricName, actual.Perc99)
+        }
+        return nil
+}
```

Comment:
We need to use tabs instead of spaces for passing pull-kubernetes-verify job as
```
diff -u ./test/e2e_node/density_test.go.orig ./test/e2e_node/density_test.go
--- ./test/e2e_node/density_test.go.orig	2019-11-27 05:38:16.360037412 +0000
+++ ./test/e2e_node/density_test.go	2019-11-27 05:38:16.360037412 +0000
@@ -536,16 +536,16 @@
 // verifyLatencyWithinThreshold verifies whether 50, 90 and 99th percentiles of a latency metric are
 // within the expected threshold.
 func verifyLatencyWithinThreshold(threshold, actual e2emetrics.LatencyMetric, metricName string) error {
-        if actual.Perc50 > threshold.Perc50 {
-                return fmt.Errorf("too high %v latency 50th percentile: %v", metricName, actual.Perc50)
-        }
-        if actual.Perc90 > threshold.Perc90 {
-                return fmt.Errorf("too high %v latency 90th percentile: %v", metricName, actual.Perc90)
-        }
-        if actual.Perc99 > threshold.Perc99 {
-                return fmt.Errorf("too high %v latency 99th percentile: %v", metricName, actual.Perc99)
-        }
-        return nil
+	if actual.Perc50 > threshold.Perc50 {
+		return fmt.Errorf("too high %v latency 50th percentile: %v", metricName, actual.Perc50)
+	}
+	if actual.Perc90 > threshold.Perc90 {
+		return fmt.Errorf("too high %v latency 90th percentile: %v", metricName, actual.Perc90)
+	}
+	if actual.Perc99 > threshold.Perc99 {
+		return fmt.Errorf("too high %v latency 99th percentile: %v", metricName, actual.Perc99)
+	}
+	return nil
 }
```